composer.json

```
@@ -17,13 +17,13 @@
         "laravel-passport-claims"
     ],
     "require": {
-        "illuminate/support": "^9.0|^10.0",
-        "laravel/passport": "^10.3|^11.2"
+        "illuminate/support": "^9.0|^10.0|^11.0",
+        "laravel/passport": "^10.3|^11.2|dev-master"
```

Passport 12 instead of dev According to the upgrade guide

```suggestion
        "laravel/passport": "^10.3|^11.2|^12.0"
```